Makefile

```diff
@@ -186,12 +186,12 @@ push-bench-images: install-bench tag-images push-images
 	docker push pachyderm/bench:`git rev-list HEAD --max-count=1`
 
 tag-images: install
-	docker tag pachyderm_pachd pachyderm/pachd:`$(GOPATH)/bin/pachctl version 2>/dev/null | grep pachctl | awk -v N=2 '{print $$N}'`
-	docker tag pachyderm_worker pachyderm/worker:`$(GOPATH)/bin/pachctl version 2>/dev/null | grep pachctl | awk -v N=2 '{print $$N}'`
+	docker tag pachyderm_pachd pachyderm/pachd:`git rev-list HEAD --max-count=1`
+	docker tag pachyderm_worker pachyderm/worker:`git rev-list HEAD --max-count=1`
```

Comment:
Does `git rev-list HEAD --max-count=1` return the same thing as `pachctl version` would in this case if it was connected. I was under the impression `pachctl version` would be something like `1.6-<commit-hash>` rather than just a commit hash.

Reply:
Yes, another solution could be to add `--porcelain` to `pachctl version` so it just spits out the client version and exits. OTOH, how likely are two different version of pachyderm to have the same commit ID? (this tag just serves to avoid docker image name collisions, right).

I'm totally happy to change `pachctl version` if that seems better, though